lib/src/application.dart

```diff
@@ -48,6 +48,11 @@ class Application {
     _writeChangelog(changelog);
   }
 
+  String set(Version version) {
+    _writeVersion(version);
+    return version.toString();
```

Comment:
What is the point in returning the argument. The caller logic has it already, right?

Reply:
Return signature is the same as `bump` method.

Reply:
Right, but bump returns something which is not known beforehand. In this case the returned value is always known ahead.